pmg/static/resources/css/style.scss

```diff
@@ -1941,3 +1941,18 @@ ul.menu {
     animation: fa-spin 1s infinite steps(16);
   }
 }
+
+.link-btn {
```

Comment:
We should be able to use Bootstrap's btn-link?

Reply:
Cool, I somehow missed that it exists. I'll use that and just style it to match the password reset text.

![screenshot at jun 21 09-45-15](https://user-images.githubusercontent.com/11753432/41704944-e2e82c76-7537-11e8-8866-70411d4fa278.png)